build-tests/api-extractor-lib2-test/etc/api-extractor-lib2-test.public.api.md

```diff
@@ -15,4 +15,6 @@ export class Lib2Class {
     prop: number;
 }
 
+export { performance_2 as performance }
```

Comment:
Note that the declaration of `performance_2` was omitted, but the de-aliased export is still included.